tests/pytests/unit/utils/vault/test_client.py

```diff
@@ -280,6 +280,18 @@ def test_vault_client_unwrap_should_default_to_token_header_before_payload(
         assert headers.get("X-Vault-Token") == token
 
 
```

Comment:
If you reset the fixture change and add the following, you shouldn't need to adapt the existing tests:

```suggestion
@pytest.mark.usefixtures("server_config")
@pytest.mark.parametrize("server_config", ({"verify": "/usr/local/share/ca-certificates/my-ca.crt"},), indirect=True)
```

Reply:
Aaaa...I need to get some more hands on with Python tests 😄 .

Fixed. Thanks!